modules/cnf-configuring-node-groups-for-the-numaresourcesoperator.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
```

Comment:
I added this as a separate procedure. However, it's also an option to just edit the existing YAML for [Creating the NUMAResourcesOperator custom resource](https://55855--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-creating-nrop-cr_numa-aware)  to include these specifications. What is the benefit of fine-tuning the polling exactly and do you think it justifies it's own topic?

Reply:
Fine-tuning is meant to be exposed as troubleshooting and/or advanced settings. The expectations is the default would be fine, and only specific conditions will require fine-tuning.

Reply:
Thanks for the info @ffromani. I think it is correct then that this is a separate procedure from the default flow. However, I'd like to include the "why" in the opening paragraph. Why would a user do this? What's the benefit? 

Something like:
"Fine-tuning the polling operations of the NROP allows you to minimize network traffic while maintaining the highest accuracy information of available NUMA resources." Or whatever the ultimate design goal is...  
WDYT?

Reply:
We anticpate fine tuning to be useful in two scenarios:
1. depending on the unpredictable combination of cluster layout (how many nodes...) and workload, improve the scheduling behavior in terms of scheduled pod throughput vs scheduler effectiveness (remove/minimize unexpected scheduling failures)
RATIONALE: we strive to provide good defaults, but "one size does not fit all" is especially true for this feature
2. debug and troubleshoot bad scheduling decisions, which is a variation of the above
3. it's indeed possible to further optimize the load between components and control plane with fine tuning, but we don't anticipate this to happen frequently yet.

In a nutshell, I'd expect fine tuning to be needed to troubleshoot failures and/or when defaults are ineffective.

Reply:
Appreciate the info @ffromani. I've added some of this to the opening paragraph. I think it gives better context to the user about when they should use this feature.